themes/custom/az_barrio/templates/layout/page.html.twig

```diff
@@ -130,9 +130,15 @@
 
           <div class="container">
             <div class="row">
-              {% if az_barrio_navbar_offcanvas %}
-              <div class="col-lg">
-                <nav class="navbar-offcanvas" id="navbarOffcanvasDemo">
+                {%
+                  set navbar_classes = [
+                    az_barrio_navbar_offcanvas ? 'navbar-offcanvas',
+                    page.navigation ? 'has-navigation-region' : 'no-navigation-region' ,
+                    page.navigation_offcanvas ? 'has-off-canvas-region' : 'no-off-canvas-region',
```

Comment:
I don't think Twig supports trailing commas.  Also a whitespace nitpick.
```suggestion
                    page.navigation ? 'has-navigation-region' : 'no-navigation-region',
                    page.navigation_offcanvas ? 'has-off-canvas-region' : 'no-off-canvas-region'
```

Reply:
Actually, the trailing comma might be fine.  I think the removal of the if statement above without removing the `endif` is what is causing the syntax error.

Reply:
@joeparsons do we want to move setting of the classes to the top of the template?

Reply:
> @joeparsons do we want to move setting of the classes to the top of the template?

Sounds reasonable to me.

Reply:
Ok. That snippet now moved up below the comments and before the opening page-wrapper.